### PR TITLE
[Autocomplete] Attempting a simpler "reset" of items

### LINKED ISSUE #1261: [LiveComponet] [Autocomplete] Bug when combining live with autocomplete select field
Hi there,

I'don't find the reason but in some cases when a field has the 'autocomplete' => true option in a live component the real value selected in the form is not the value sent when submitting.

I've created a repository to reproduce the bug:
 [https://github.com/tdumalin/bug-ux-live-plus-autocomplete](https://github.com/tdumalin/bug-ux-live-plus-autocomplete)
All the installation informations are setted in the README file.

I've notice the bug multiple times but always on select input, noticed on:
ChoiceType
EnumType
EntityType

Adding a random data-live-id on the row_attr seems to resolve the bug but it's not convenient to set globally
```twig
{{ form_row(form.type, {row_attr: {
        'data-live-id': random()
}}) }}
```
If needed here is my work around:

```php
<?php

namespace App\Form\Extension;

use Symfony\Bridge\Doctrine\Form\Type\EntityType;
use Symfony\Component\Form\AbstractTypeExtension;
use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
use Symfony\Component\Form\Extension\Core\Type\EnumType;
use Symfony\Component\OptionsResolver\Options;
use Symfony\Component\OptionsResolver\OptionsResolver;

class AddAutocompleteExtension extends AbstractTypeExtension
{

    public function configureOptions(OptionsResolver $resolver): void
    {
        $resolver->setDefaults([
            'autocomplete' => true,
        ]);


        $resolver->setNormalizer('row_attr', function (Options $options, $value) {
            if ($options['autocomplete']) {
                $value['data-live-id'] = uniqid('autocomplete-');
            }

            return $value;
        });
    }

    public static function getExtendedTypes(): iterable
    {
        return [
            ChoiceType::class,
            EntityType::class,
            EnumType::class,
        ];
    }
}

```

NB: This work around making you lost the focus of the input

### PR DESCRIPTION
**Pull request by weaverryan**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| Issues        | Possibly #1290 (need to check and remove data-live-ignore), fixes #1261 possibly #1241 possibly #909
| License       | MIT

Tomselect is a real pain in the butt. When you select an option, it re-orders the `option` elements and uses the elements themselves internally to remember which one is selected. It plays poorly with LiveComponents, which is why we added the MutationObserver logic. However, at least in one straightforward case that I had today, when you selected a new item, the new value was being lost. This will only get worse when Turbo 8 adds morphing.

This PR attempts to simplify: when needed, completely destroy TomSelect and recreate it. It seems to work well, but 2 tests are failing. And I'm pulling my hair out - so it may not even be fixable. No matter what I try, when I reinitialize TomSelect on a set of HTML that has been reordered, it gets confused. It's as if it has a static cache somewhere that I can't find...

### CODE REVIEW

**Comment by weaverryan** [comment]
Related: https://github.com/orchidjs/tom-select/issues/666

I simply can't figure out how to make TomSelect keep the selected option value.

**Comment by weaverryan** [comment]
SO happy to say that this is ready 😅.

tl;dr TomSelect rearranges the `<option>` elements. If something "morphes" the `<option>` element like LiveComponents (or Turbo 8) does, TomSelect can't handle this. This PR removes morphing for the`<option>` elements for Autocomplete and fixes & improves how we reinitialize TomSelect if the option elements DO change.
